src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -62,6 +62,7 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Redis.Tests" />
```

Comment:
We should aim to remove this

Reply:
Do we have an issue for tracking that?

Reply:
We dont

Reply:
This one seems worse. We shouldn't be using internal stuff from `Aspire.Hosting` in our extension library's tests. What is this needed for?

Reply:
I removed it and it's fine now.

Reply:
Also removed the other one